### PR TITLE
[18.0][MIG] document_page: Migration to 18.0

### PR DESCRIPTION
**Pull request by jdoutreloux**
Depends on https://github.com/OCA/knowledge/pull/503

### CODE REVIEW

File: document_page/views/document_page.xml
Comment (natuan9) [inline document_page/views/document_page.xml]: It seems like you forgot to replace this with `<chatter />`

Comment (jdoutreloux) [reply document_page/views/document_page.xml]: Thanks for the catch. It's fixed now

**Comment by TDu** [comment]
@jdoutreloux  Thanks for the migration it looks good.

To fix the tests and runboat, you need to mention the dependency because it is not merged yet, following this https://github.com/OCA/maintainer-tools/wiki/Use-temporary-reference(s)-to-another-pull-request(s)

**Review by TDu — APPROVED** [review]

**Review by gbrito — APPROVED** [review]

**Review by natuan9 — APPROVED** [review]

**Comment by TDu** [comment]
@jdoutreloux #503 has been merged could you drop the commit [[DON'T MERGE] test-requirements.txt](https://github.com/OCA/knowledge/pull/504/commits/86803a366e2c8b88ce3dc2d967e32ac1f9fa971e)
Thanks

**Comment by natuan9** [comment]
There are several PRs from 14.0 that are not included in 17.0.
These PR port those to 17.0 and 18.0:
- https://github.com/OCA/knowledge/pull/510
- https://github.com/OCA/knowledge/pull/511

**Comment by natuan9** [comment]
Hi @jdoutreloux ,
There are several PRs in 14.0 that are not included in 17.0. I created PR to port them to 17.0
- https://github.com/OCA/knowledge/pull/510

Now, I create this PR to your branch for migration to 18.0
- https://github.com/jdoutreloux/knowledge/pull/1

**Comment by jdoutreloux** [comment]
> @jdoutreloux #503 has been merged could you drop the commit [[DON'T MERGE] test-requirements.txt](https://github.com/OCA/knowledge/pull/504/commits/86803a366e2c8b88ce3dc2d967e32ac1f9fa971e) Thanks

I've removed the commit

**Pull request merged**
